io/pci/pci_hotplug.py.data/pci_hotplug.yaml

```diff
@@ -1,3 +1,5 @@
 pci_devices: ""
 count: 10
+#This option is for nvme splitter adapter, Value: nvme_splitter
+adapter_type: ""
```

Comment:
is this for all nvme types ? or only splitter

Reply:
This is only for splitter